mlperf_logging/rcp_checker/training_3.0.0/rcps_gpt3.json

```diff
@@ -0,0 +1,78 @@
+{
+
+  "gpt3_ref_1536":
+  {
+    "Benchmark": "gpt3",
+    "Creator": "Google & NVIDIA",
+    "When": "Prior to 3.0 submission",
+    "Platform": "TPU-v4-1536 / PaxML, 64 A100-80GB / Megatron-LM",
```

Comment:
1024 A100-80GB GPUs were used for all RCPs

Reply:
Thanks for the info. Updated the PR.